client/state/reader/start/reducer.js

```diff
@@ -51,6 +54,29 @@ export function items( state = [], action ) {
 }
 
 /**
+ * Returns the updated "graduation" from cold start request state after an
+ * action has been dispatched.
+ *
+ * @param  {Object} state  Current state
+ * @param  {Object} action Action payload
+ * @return {Object}        Updated state
+ */
+export function isRequestingGraduation( state = false, action ) {
```

Comment:
usually we use that to prevent duplicate inflight requests, often in a query component. How does this thing get invoked? Do we do anything to prevent dupes?

This is a bit of an odd one as the action is idempotent, we're only setting it one way... So likely not actually necessary to dedupe.. hrm.